packages/glimmer-syntax/lib/parser.ts

```diff
@@ -38,16 +38,60 @@ export function preprocess(html, options?) {
 
 const entityParser = new EntityParser(namedCharRefs);
 
-export function Parser(source, options) {
-  this.options = options || {};
-  this.elementStack = [];
-  this.tokenizer = new EventedTokenizer(this, entityParser);
+export class Parser {
+  private currentAttribute = null;
+  private currentNode = null;
```

Comment:
Wondering if I can remove the above two properties. Can't find where they're used anywhere, internally or externally.